src/main/java/com/djrapitops/extension/GriefDefenderExtension.java

```diff
@@ -64,11 +69,10 @@ public GriefDefenderExtension() {
     }
 
     private List<Claim> getClaimsOf(UUID playerUUID) {
-        return GriefDefender.getCore().getAllPlayerClaims(playerUUID);
+        return api.getAllPlayerClaims(playerUUID);
     }
-
-    private String formatLocation(Vector3i greaterBoundaryCorner) {
-        return "x: " + greaterBoundaryCorner.getX() + " z: " + greaterBoundaryCorner.getZ();
+    private PlayerData getPlayerData(UUID playerUUID) {
+        return Objects.requireNonNull(api.getUser(playerUUID)).getPlayerData();
```

Comment:
Objects.requireNonNull will throw a NullPointerException if the value is null - consider throwing `new NotReadyException()` instead if the user is null since Plan handles that one better.